exporters/trace/jaeger/src/test/java/io/opencensus/exporter/trace/jaeger/JaegerExporterHandlerTest.java

```diff
@@ -142,6 +144,44 @@ public void exportShouldConvertFromSpanDataToJaegerThriftSpan() throws SenderExc
     assertThat(reference.refType).isEqualTo(SpanRefType.CHILD_OF);
   }
 
+  @Test
+  public void convertErrorSpanDataToJaegerThriftSpan() throws SenderException {
+    long startTime = 1519629870001L;
+    long endTime = 1519630148002L;
+    String statusMessage = "timeout";
+    SpanData spanData =
+        SpanData.create(
+            sampleSpanContext(),
+            SpanId.fromBytes(new byte[] {(byte) 0x7F, FF, FF, FF, FF, FF, FF, FF}),
+            true,
+            "test",
+            Kind.SERVER,
+            Timestamp.fromMillis(startTime),
+            SpanData.Attributes.create(Collections.<String, AttributeValue>emptyMap(), 0),
+            SpanData.TimedEvents.create(Collections.<TimedEvent<Annotation>>emptyList(), 0),
+            SpanData.TimedEvents.create(Collections.<TimedEvent<MessageEvent>>emptyList(), 0),
+            SpanData.Links.create(Collections.<Link>emptyList(), 0),
+            0,
+            Status.DEADLINE_EXCEEDED.withDescription(statusMessage),
+            Timestamp.fromMillis(endTime));
+
+    handler.export(singletonList(spanData));
+
+    verify(mockSender).send(eq(process), captor.capture());
+    List<Span> spans = captor.getValue();
+
+    assertThat(spans.size()).isEqualTo(1);
+    Span span = spans.get(0);
+
+    assertThat(span.tags.size()).isEqualTo(4);
+    assertThat(span.tags)
+        .containsExactly(
+            new Tag("span.kind", TagType.STRING).setVStr("server"),
+            new Tag("status.code", TagType.LONG).setVLong(4),
+            new Tag("status.message", TagType.STRING).setVStr(statusMessage),
+            new Tag("error", TagType.BOOL).setVBool(true));
```

Comment:
nit:s/status.code/STATUS_CODE/ 
same with other strings.

Reply:
SG, fixed in https://github.com/census-instrumentation/opencensus-java/pull/1928/commits/c7816a7e16802ab835f5eeb6ce471ae863316bc1.